recipes/elfio/all/conandata.yml

```diff
@@ -0,0 +1,6 @@
+sources:
+  "3.8":
+    sha256: 9553ce2b8d8aa2fb43f0e9be9bcbd10cd52f40b385110ea54173889c982f9ac4
+    url: https://github.com/serge1/ELFIO/releases/download/Release_3.8/elfio-3.8.tar.gz
+
+
```

Comment:
```suggestion
    url: https://github.com/serge1/ELFIO/releases/download/Release_3.8/elfio-3.8.tar.gz
```

Reply:
I don't see a diff, whitespace at the end?  
or the empty line (think one is required)

Reply:
Yes its the whitespace

Reply:
strange, my editor config should prevent that ....

Reply:
no, there are no change, when I try the Commit suggestion button, it says there are noch changes ...

Reply:
It'll get fix when the next version gets add... I'd rather see it merged 👾